orchagent/interface_rates.lua

```diff
@@ -0,0 +1,95 @@
+-- KEYS - rif IDs
+-- ARGV[1] - poll time interval
+-- return nothing 
+
+local asic_db = 1 
+local counters_db = 2
+local config_db = 4 
+-- local fc_db = 5
+local counters_table_name = "COUNTERS" 
+-- local asic_table_name = "ASIC_STATE:SAI_OBJECT_TYPE_PORT" 
+local rates_table_name = "RATES"
+-- local fc_group_table_name = "FLEX_COUNTER_GROUP_TABLE:PORT_STAT_COUNTER"
+
+local rets = {}
+
+-- Get configuration
+redis.call('SELECT', config_db)
+local smooth_interval = redis.call('GET', rates_table_name .. ':' .. 'RIF_SMOOTH_INTERVAL')
+local alpha = redis.call('GET', rates_table_name .. ':' .. 'RIF_ALPHA')
+local one_minus_alpha = 1 - alpha
+local delta = ARGV[1]
```

Comment:
Looks like ARGV[1] is counters DB index https://github.com/mykolaf/sonic-swss/blob/master/orchagent/pfc_detect_mellanox.lua#L2